.github/main.workflow

```diff
@@ -7,6 +7,11 @@ action "Trigger GitHub release" {
   uses = "./.github/actions/trigger-github-release/"
   secrets = ["GITHUB_TOKEN"]
 
+  env = {
+    COMMIT_AUTHOR_NAME  = "Babel Bot"
+    COMMIT_AUTHOR_EMAIL = "babel@hopeinsource.com"
```

Comment:
shoudln't this be bot@babeljs.io?

Reply:
I don't know if it exists? `babel@hopeinsource.com` is the mail in the @babel-bot account config, so it works to link the commit to that account.